Makefile

```diff
@@ -103,7 +103,11 @@ clean-schema:
 
 # tests for go packages
 test-gopkgs:
-	$(Q)$(GO_TEST) ./...
+	$(Q)for mod in $$(find . -name go.mod); do \
+	    echo "Testing $$(dirname $$mod)..."; ( \
+	        cd $$(dirname $$mod) && $(GO_TEST) ./... \
+            ) || exit 1; \
```

Comment:
Would it exit on the first test failure? If so, could it be modified to run all tests even if one fails?

Reply:
I have updated this to run all tests and exit on failures afterwards.